config/default.json

```diff
@@ -1,6 +1,6 @@
 {
     "server": {
-        "port": 3000
+        "port": 3009
```

Comment:
check if this file need to be removed and update the conf.d file

Reply:
fixed in 56b66c